2.js

```diff
@@ -0,0 +1,86 @@
+const readline = require('readline')
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+const isValidTile = character => {
+  if (typeof character !== 'string') return false
+
+  const characterCode = character.charCodeAt(0)
+
+  if (
+    (characterCode >= 65 && characterCode <= 90)
```

Comment:
Whenever I see the pattern:
```js
if( booleanExpression ) {
  return true
} else {
  return false
}
```
I replace it with a return statement that returns the boolean expression itself:
```js
return ( characterCode >= 65 && characterCode <= 90 ) || characterCode === 95
```
You might also consider using constants to replace the "magic numbers" in the code here:
```js
return ( characterCode >= CHARACTER_A && characterCode <= CHARACTER_Z ) || 
  characterCode === CHARACTER_UNDERSCORE
```